pystac/extensions/single_file_stac.py

```diff
@@ -1,137 +0,0 @@
-from pystac.catalog import Catalog
```

Comment:
Was it intentional to remove this extension?

Reply:
It was - there's upcoming changes the extension that make it being based off of a Catalog basically incorrect, and there's issue with it's usage (e.g. #256). It felt like it needed rework to be better in this PR as part of the upgrade, but didn't seem worth the effort given that it'll change. Perhaps we want to re-add in a follow up PR that is coordinated with the work to get ItemCollection in (#302) and moves the single-file-stac extension to being based on the ItemCollection (or is removed completely in favor of ItemCollection)

Reply:
Thanks, that all makes sense.